arch/arm64/boot/dts/rockchip/rk3588-blueberry-edge-v10.dtsi

```diff
@@ -397,9 +398,9 @@
 		adc-volume = <0xbf>;    /* 0dB */
 		dac-volume = <0xbf>;    /* 0dB */
 		aec-mode = "adc left, adc right";
-		clocks = <&mclkout_i2s0>;
+		clocks = <&cru I2S0_8CH_MCLKOUT>;
```

Comment:
Does this change make audio works?

Reply:
Take a look at this commit: https://github.com/armbian/linux-rockchip/commit/005807f776ef16a62e58b32cb7db9991d6b189f4

Reply:
Super thanks, I had missed it, I will preserve your changes in the next push.

Reply:
Interesting that the `v12` from vendor already had those. I re-picked 005807f for the `v10`